src/test/resources/pretty_printed_xml.xml

```diff
@@ -1,66 +1,60 @@
-            <orx:drift/>
-            <cc:location/>
-          </orx:meta>
-        </data>
-      </instance>
-      <instance id="commcaresession" src="jr://instance/session"/>
-      <bind nodeset="/data/q_name" required="true()" type="xsd:string"/>
-      <itext>
-        <translation default="" lang="en">
-          <text id="q_name-label">
-            <value>Enter a name:</value>
-            <value form="image">jr://file/commcare/image/data/q_name-25t9zc.png</value>
-            <value form="long">long</value>
-            <value form="short">short</value>
-          </text>
-        </translation>
-        <translation lang="hin">
-          <text id="q_name-label">
-            <value>HIN: Enter a name:</value>
-            <value form="image">jr://file/commcare/image/data/q_name-25t9zc.png</value>
-            <value form="long">long</value>
-            <value form="short">short</value>
-          </text>
-        </translation>
-      </itext>
-      <setvalue event="xforms-ready" ref="/data/meta/deviceID" value="instance('commcaresession')/session/context/deviceid"/>
-      <setvalue event="xforms-ready" ref="/data/meta/timeStart" value="now()"/>
-      <bind nodeset="/data/meta/timeStart" type="xsd:dateTime"/>
-      <setvalue event="xforms-revalidate" ref="/data/meta/timeEnd" value="now()"/>
-      <bind nodeset="/data/meta/timeEnd" type="xsd:dateTime"/>
-      <setvalue event="xforms-ready" ref="/data/meta/username" value="instance('commcaresession')/session/context/username"/>
-      <setvalue event="xforms-ready" ref="/data/meta/userID" value="instance('commcaresession')/session/context/userid"/>
-      <setvalue event="xforms-ready" ref="/data/meta/instanceID" value="uuid()"/>
-      <setvalue event="xforms-ready" ref="/data/meta/appVersion" value="instance('commcaresession')/session/context/appversion"/>
-      <setvalue event="xforms-revalidate" ref="/data/meta/drift" value="if(count(instance('commcaresession')/session/context/drift) = 1, instance('commcaresession')/session/context/drift, '')"/>
-      <orx:pollsensor event="xforms-ready" ref="/data/meta/location"/>
-      <bind nodeset="/data/meta/location" type="geopoint"/>
-    </model>
-  </h:head>
-  <h:body>
-    <input ref="/data/q_name">
-      <label ref="jr:itext('q_name-label')"/>
-    </input>
-  </h:body>
+<?xml version="1.0" encoding="UTF-8"?><h:html xmlns:h="http://www.w3.org/1999/xhtml" xmlns:jr="http://openrosa.org/javarosa" xmlns:orx="http://openrosa.org/jr/xforms" xmlns:vellum="http://commcarehq.org/xforms/vellum" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+    <h:head>
+        <h:title>Basic Form</h:title>
+        <model>
+            <instance>
+                <data name="Basic Form" uiVersion="1" version="1" xmlns:jrm="http://dev.commcarehq.org/jr/xforms">
+                    <q_name/>
+                    <orx:meta xmlns:cc="http://commcarehq.org/xforms">
+                        <orx:deviceID/>
+                        <orx:timeStart/>
```

Comment:
There's a change in the indentation space with `LSSerializer`. I couldn't find any api that could let me change the indent amount. 
There's another way to use `TransformerFactory` which has a parameter `indent-amount` to change the indentation amount. 
Should I use that or is it fine?

Reply:
should be fine for indentation to change.